src/frontend/components/registrationForm.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from "react";
+import Input from "./input";
+import Button from "./button";
+import Select from "./Select";
+
+class RegistrationForm extends Component {
+  constructor(props) {
+    super(props);
+  }
+  handleChange = () => {
+    console.log("Changes made in input ");
+  };
+  handleSubmit = () => {
+    console.log("Submit button clicked");
+  };
+  getDays = () => {
+    const days = [];
+    for (let i = 1; i <= 31; i++) {
+      days.push(i);
+    }
+    return days;
+  };
+  getMonths = () => {
+    var months = [
```

Comment:
Please learn to be consistent with the use of es6 syntax. Instead of `var`, opt for the use of `let` and `const`. Also, in this particular function, You could just return the array without wasting memory. Another point of consideration is that, instead of creating a new array every time this method is called, you could consider making it cacheable (shareable) so that the same array is presented instead of creating a new copy always. You can consider creating this as a property of the class.